### ISSUE TITLE
Huge JSON

### ISSUE THREAD

**Issue by SqrTT** [dulnan/nuxt-multi-cache#21]
`setItem` serializes passed value to JSON string. Also afterwards `getItem` use `JSON.parse` to de-serialize value back.

In cases when page size is big, say, 1-2 Megabytes this has significant effect on performance. `JSON.parse` cause high CPU load and it's event blocking (so server stop doing other stuff like accept incoming requests)

As possible solution is use two records one for page itself other for meta (statusCodes, headers)
page content may by stored by using `getItemRaw/setItemRaw` which don't do serialization to JSON.

Also is possible to use single record but avoid usage JSON parse and implement own lightweight serialization, ie store record in next way format: `{statusCode: 200, headers: []}<DELIMETER><html>...</html>`
and extract small JSON
```js
const delimiterPos = record.indexOf('<DELIMETER>')
const meta = JSON.parse(record.substr(0,delimiterPos))
const data = record.substr(delimiterPos)
``` 
So we may avoid parsing huge JSONs

**Comment by dulnan** [comment]
Thank you for this suggestion, this seems to be a new feature in unstorage, or at least it wasn't present when I released the first Nuxt 3 version. For the route cache it indeed makes absolute sense, as well as component cache. For the data cache, I think it also makes sense, but it would only be used if calling addToCache with a string.

**Comment by dulnan** [comment]
This is implemented in the 3.1.0 release. While setItemRaw is still experimental, it seemed to work as expected. So I think we should be fine.

**Comment by SqrTT** [comment]
that's awesome. Thank you!
